packages/expo-image-picker/src/ImagePicker.types.ts

```diff
@@ -56,6 +65,7 @@ export type ImagePickerOptions = {
   exif?: boolean;
   base64?: boolean;
   videoExportPreset?: VideoExportPreset;
```

Comment:
```suggestion
	/**
    * @deprecated use `videoQuality` instead.
    */
  videoExportPreset?: VideoExportPreset;
```

Reply:
I might be wrong but based on the documentation it seems videoQuality is only effecting **recording**, meanwhile `videoExportPreset` is also doing a conversion when picking a video from the gallery, so it's not a 1 to 1 mapping.

Reply:
So it will be a better idea to leave these two options 😅
Please remove `deprecated` word from the documentation too ;)

Reply:
it does not change the fact that it's deprecated :) 
https://developer.apple.com/documentation/uikit/uiimagepickercontroller/2890964-videoexportpreset?language=objc

Reply:
Sorry, I forgot about it 😅  My bad.

Reply:
So add a comment that `videoExportPreset` is deprecated, but don't redirect a user to use `videoQuality` 😉
You can link the apple documentation int this comment.